pkg/cmd/utils/viper.go

```diff
@@ -72,6 +74,12 @@ func AddDefaultFlagCompletion() cmd.Option {
 	}
 }
 
+//ServetIpKey returns the key value of ConfigServerIP
+//Avoids having to constants public
+func ServerIpKey() string {
+	return configServerIP
+}
+
```

Comment:
Is this used?
also "avoids having to constants public" doesn't make much sense.